### DISCUSSION TITLE
Games using Gorgon

### DISCUSSION THREAD

**Discussion by Trilarion** [Tape-Worm/Gorgon#48] [General]
Hi,

Are there any games (at least playable) that are using Gorgon? Maybe this discussion could contain a list of these.

**Comment by Tape-Worm** [comment]
Unfortunately, I don't know of any offhand. I have my doubts that many people actually use this monstrosity that I've built. 

If you're just interested in seeing what the APIs are capable of, you can try this little experimental thing I wrote ages back:
[https://www.tape-worm.net/public/PewPew.zip](https://www.tape-worm.net/public/PewPew.zip)

And these are the instructions for "playing" it:
[https://www.tape-worm.net/public/Read%20me.txt](https://www.tape-worm.net/public/Read%20me.txt)

Just unzip and run.  If you don't have .NET 6 installed (which by now you probably do), then get it from here:
[https://dotnet.microsoft.com/en-us/download/dotnet/6.0](https://dotnet.microsoft.com/en-us/download/dotnet/6.0) (Windows Desktop, x64)

**Comment by Trilarion** [comment]
Thanks for the reply. So PewPew is basically something like a tech demo. It's still good to have because it shows what can be done with the game engine.

**Reply by Tape-Worm** [reply]
More or less, but it was never really meant for public consumption.

I have a more comprehensive suite of demo applications that displays what can be accomplished with the API.  I had assumed you already saw that.

Unlike what I sent you, these all have source code available in the Gorgon project.

The binaries for the demos are on the Releases page:
[https://github.com/Tape-Worm/Gorgon/releases/download/v3.2.26.144/Gorgon_Examples_3.2.26.144.zip](https://github.com/Tape-Worm/Gorgon/releases/download/v3.2.26.144/Gorgon_Examples_3.2.26.144.zip)

The source code for the examples is located here:
[https://github.com/Tape-Worm/Gorgon/tree/main/Examples](https://github.com/Tape-Worm/Gorgon/tree/main/Examples)
